--- a/Makefile
+++ b/Makefile
@@ -255,18 +255,12 @@
 	mkdir -p ${DEST_LIB}/pcs/
 	mv ${DEST_PYTHON_SCRIPT_DIR}/pcs_internal ${DEST_LIB}/pcs/pcs_internal
 	# pcs SNMP install
-	mv ${DEST_PYTHON_SCRIPT_DIR}/pcs_snmp_agent ${DEST_LIB}/pcs/pcs_snmp_agent
-	install -d ${DEST_SNMP_MIB}
-	install -m 644 pcs/snmp/mibs/PCMK-PCS*-MIB.txt ${DEST_SNMP_MIB}
-	install -m 644 -D pcs/snmp/pcs_snmp_agent.conf ${DEST_CONF}/pcs_snmp_agent
-	install -m 644 -D pcs/snmp/pcs_snmp_agent.8 ${DEST_MAN}/pcs_snmp_agent.8
 ifeq ($(IS_DEBIAN),true)
 	$(call use-alternative-file,pcs/settings.py.debian,${DEST_PYTHON_SITELIB}/pcs/settings.py)
 endif
 	$(PYTHON) -m compileall -fl ${DEST_PYTHON_SITELIB}/pcs/settings.py
 ifeq ($(IS_SYSTEMCTL),true)
 	install -d ${DEST_SYSTEMD_SYSTEM}
-	install -m 644 pcs/snmp/pcs_snmp_agent.service ${DEST_SYSTEMD_SYSTEM}
 endif
 
 install: install_python_part
